diff --git a/termite.cc b/termite.cc
index e168e77..993acca 100644
--- a/termite.cc
+++ b/termite.cc
@@ -1047,8 +1047,7 @@ gboolean key_press_cb(VteTerminal *vte, GdkEventKey *event, keybind_info *info)
             case GDK_KEY_KP_Add:
                 increase_font_scale(vte);
                 return TRUE;
-            case GDK_KEY_minus:
-            case GDK_KEY_KP_Subtract:
+            case GDK_KEY_underscore:
                 decrease_font_scale(vte);
                 return TRUE;
             case GDK_KEY_equal:
